_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -67,6 +105,12 @@ class HealthCareApp extends React.Component {
     this.setState({ applicationData: newApplicationData });
   }
 
+  handleButtonClick() {
+    const nextPath = this.getNextUrl();
```

Comment:
Don't bother with the variable definition if the expression is short and there's only one use. Just do hashHistory.push(this.getNextUrl());